hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/customfees/FcmCustomFeeSchedules.java

```diff
@@ -30,20 +34,35 @@
 @Singleton
 public class FcmCustomFeeSchedules implements CustomFeeSchedules {
     private final Supplier<MerkleMap<EntityNum, MerkleToken>> tokens;
+    private TransactionalLedger<TokenID, TokenProperty, MerkleToken> tokensLedger;
```

Comment:
Is there a reason to not just do a separate implementation that always uses a `tokensLedger`?

Reply:
DONE